examples/tools/gamepad_viewer.rs

```diff
@@ -341,7 +341,7 @@ fn setup_sticks(
                                 style,
                             },
                         ])
-                        .with_alignment(TextAlignment::BOTTOM_CENTER),
+                        .with_alignment(TextAlignment::Center),
```

Comment:
The text here would no longer be "bottom center" unless there also is a corresponding change to the parent layouting.

All examples should look and feel exactly the same before and after this change.

Reply:
The odd, unintuitive thing I only just realised is that for Text2d, TextAlignment doesn't do anything now when used with single-line text. Using bevy_sprite's anchor type makes sense, kind of, because it's also used the same way with Sprites and the text is drawn with sprites. It's all correct, but not entirely pleasant.

Reply:
Maybe TextAlignment should be renamed to TextJustification but that seems strange without support for fully justified text.

Reply:
Yep, that change can wait.